electron/index.js

```diff
@@ -287,10 +286,14 @@ async function performFsJob(job) {
     }
 }
 
-ipcMain.on("fs-job", async (event, arg) => {
-    const result = await performFsJob(arg);
-    event.reply("fs-response", { id: arg.id, result });
-});
+ipcMain.handle(
+    "fs-job",
+    (event, arg) =>
+        new Promise(async (resolve, reject) => {
```

Comment:
An async method is a promise automatically, so why not do `(event, arg) => performFsJob(arg)` ?

Reply:
Done that.